ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -9873,6 +9873,50 @@ def test_torch_tensor_quantile(
     )
 
 
+# random_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="random_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=["float16", "float32", "float64", "int32", "int64"],
```

Comment:
This is not the ideal way of dealing with unsupported data dtypes. you should add `unsupported_data_decorator` at the top of the function, otherwise, even though tests are passing but realistic sense, there's no restriction for `bfloat`
thanks!